packages/snjs/lib/models/app/search.ts

```diff
@@ -0,0 +1,40 @@
+/** Grouping of required state variables to perform the search */
+
+import { CollectionSort } from "@Lib/protocol/collection/item_collection";
+import { SNTag } from "./tag";
+
+export interface SearchPayload {
+  tag: SNTag,
+  sortBy: CollectionSort,
+  isReversedSort: "asc" | "dsc" | undefined,
```

Comment:
If you want to communicate an argument being optional (instead of explicitly `undefined`), I think `isReversedSort?: "asc" | "dsc"` would communicate that intent clearer :) https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties

Reply:
Should also use single quotes instead of double quotes.

Reply:
Done. Should I use single quotes in the future for all strings?